libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/NavigationRoute.kt

```diff
@@ -764,6 +775,10 @@ internal constructor(
             return waypoints
         }
 
+        @NonNull
```

Comment:
`@NonNull` is redundant.
also radiuses might be `null` so return value `List<Double?>`

Reply:
Great catch!

Reply:
👍